test/python/_mockutils.py

```diff
@@ -30,12 +30,17 @@
 
 class DummyProvider(BaseProvider):
     """Dummy provider just for testing purposes."""
+    def __init__(self):
+        self._backend = DummySimulator()
+
+        super(DummyProvider, self).__init__()
```

Comment:
I'm unsure but you can do `super().__init__()` in Python 3, can't you?

Reply:
Yes - it is actually the recommended way in our project, thanks for catching it! :+1: